core/src/main/java/de/valtech/aecu/core/groovy/console/bindings/actions/resource/RemoveMixin.java

```diff
@@ -0,0 +1,49 @@
+package de.valtech.aecu.core.groovy.console.bindings.actions.resource;
+
+import de.valtech.aecu.core.groovy.console.bindings.actions.Action;
+import de.valtech.aecu.core.groovy.console.bindings.actions.util.MixinUtil;
+import javax.annotation.Nonnull;
+import javax.jcr.Node;
+import javax.jcr.RepositoryException;
+import javax.jcr.nodetype.NoSuchNodeTypeException;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.sling.api.resource.PersistenceException;
+import org.apache.sling.api.resource.Resource;
+
+/**
+ * Removes a mixin from a resource/node if the mixin is present.
+ *
+ * @author Eric Manzi and Bart Thierens
+ */
+public class RemoveMixin implements Action {
+
+    private final String mixinName;
+
+    public RemoveMixin(String mixinName) {
+        this.mixinName = mixinName;
+    }
+
+    @Override
+    public String doAction(@Nonnull Resource resource) throws PersistenceException {
+        if (StringUtils.isBlank(mixinName)) {
+            return "WARNING: mixin name is empty";
+        }
+        Node node = resource.adaptTo(Node.class);
+        if (node == null) {
+            return "WARNING: could not get node for " + resource.getPath();
+        }
+
+        try {
+            if (MixinUtil.hasMixin(node, mixinName)) {
+                node.removeMixin(mixinName);
```

Comment:
the `context.isDryRun()` logic would need to be implemented.

Reply:
same remark as for addMixin, the change is not persisted for a dry-run